src/control/drpc/modules.go

```diff
@@ -0,0 +1,76 @@
+//
+// (C) Copyright 2019 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+// This file imports all of the DAOS dRPC module/method IDs.
+
+package drpc
+
+// #cgo CFLAGS: -I${SRCDIR}/../../include
+// #include <daos/drpc_modules.h>
+import "C"
+
+const (
+	// ModuleSecurityAgent is the dRPC module for security tasks in the
+	// DAOS agent
+	ModuleSecurityAgent = C.DRPC_MODULE_SEC_AGENT
+	// MethodRequestCredentials is a ModuleSecurityAgent method
+	MethodRequestCredentials = C.DRPC_METHOD_SEC_AGENT_REQUEST_CREDS
+
+	// ModuleMgmt is the dRPC module for management service tasks
+	ModuleMgmt = C.DRPC_MODULE_MGMT
+	// MethodKillRank is a ModuleMgmt method
+	MethodKillRank = C.DRPC_METHOD_MGMT_KILL_RANK
+	// MethodSetRank is a ModuleMgmt method
+	MethodSetRank = C.DRPC_METHOD_MGMT_SET_RANK
```

Comment:
Not worth blocking this on, but if you have to make any more adjustments I'd recommend breaking these consts into groups, e.g. Module*, Method*, etc.

Reply:
I will definitely need to do a bit more work on this after #1229 lands - there will be conflicts, so I can address it at that point.

Reply:
Fixed the grouping